Javascript-1/script.js

```diff
@@ -0,0 +1,31 @@
+// Build Specs
+
+const firstname="Karina";
```

Comment:
Remeber to use camel case when naming variables. This const should be named firstName